test/parallel/test-http-hostname-typechecking.js

```diff
@@ -21,7 +21,7 @@ vals.forEach((v) => {
 // These values are OK and should not throw synchronously
 ['', undefined, null].forEach((v) => {
   assert.doesNotThrow(() => {
-    http.request({hostname: v}).on('error', common.noop).end();
-    http.request({host: v}).on('error', common.noop).end();
+    http.request({hostname: v}).on('error', common.mustCall()).end();
+    http.request({host: v}).on('error', common.mustCall()).end();
```

Comment:
(Oh! Reverting this one won't work because common.noop is no longer A Thing. So #14065 is probably the closest thing to a revert.)